Sources/Tests/EndToEnd/Test_EndToEnd.c

```diff
@@ -72,7 +72,6 @@ static void ReadTemplate(const char *expectedFileName, Template *expectedTemplat
     assert(ret != EOF);
 }
 
-
```

Comment:
@david-connell Thanks!  I guess I'd love to disable that malloc stuff, since valgrind does that job for us.  

Most of the things I've mentioned are library related (`const`) or unrelated to this (use of unity), so I'm happy for this to be merged!